samples/nodejs/EchoBot/README.md

```diff
@@ -0,0 +1,69 @@
+# Echo-bot
+
+This is a sample of a simple Agent that is hosted on an Node.js web service.  This Agent is configured to accept a request and echo the text of the request back to the caller.
+
+This Agent Sample is intended to introduce you to the basic operation of the Microsoft 365 Agents SDK messaging loop. It can also be used as the base for a custom Agent you choose to develop.
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org) version 18 or higher
+
+    ```bash
+    # determine node version
+    node --version
+    ```
+- [dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows)
+- [Bot Framework Emulator](https://github.com/Microsoft/BotFramework-Emulator/releases) for Testing Web Chat.
```

Comment:
Emulator is not a requirement. For non-auth we can show how to use the teams test app tool